src/omero/plugins/render.py

```diff
@@ -241,11 +240,11 @@ def to_dict(self):
         # self.range
         # self.model
         # self.projection
-        # self.invertAxis
         return d
 
     def close(self):
-        self.image._closeRE()
+        print 'not yet implemented'
+        # self.image._closeRE()
```

Comment:
Should this be implemented?
e.g.
```
if self.image._re is not None:
    try:
        self.image._re.close()
    except:
         # log...

Reply:
I could to that in the meantime
until the method is available in the main line